Readability.js

```diff
@@ -1743,12 +1743,15 @@ Readability.prototype = {
         return false;
       }
 
-      var textContentLength = node.textContent.trim().length;
-      if (textContentLength < 140) {
-        return false;
+      // In some web pages, they use "<br>" to typeset articles.
+      if (node.tagName === "BR") {
+        score += 2;
```

Comment:
Yes, there is no cheap way to deal with all kinds of web pages made by the site users. There is a lot of situations.

After investigated the blogger articles, I found out many articles'[1], [2], [3], [4], [5], [6], [7], [8], [9], [10](random pages from 10 bloggers) DOM structures look like
```
<div>
  Sentences<br>
  <br>
  Sentences<br>
  <br>
  Sentences<br>
  <br>
  Sentences<br>
  <br>
</div>
``` 

So how about we focus this pattern of DOM structure first? If OK, we can try to implement a light-weight algorithm to detect the pattern.

Or do we prefer to implement an algorithm to deal with almost all kinds of web page structure made by blogger users? If so, I think we should try to list all kinds of DOM structures look like articles first. Then we can implement an algorithm to detect them.

What do you think, @gijsk ? :)

[1]: http://siliconexposed.blogspot.tw/2016/05/open-verilog-flow-for-silego-greenpak4.html
[2]: http://www.stevenhoneyman.co.uk/2016/02/bga-on-budget.html
[3]: http://sequence15.blogspot.tw/2015/11/review-ssl-1250-quad-lfo.html
[4]: http://teslaprime.blogspot.tw/2016/04/the-behavior-of-capacitor.html
[5]: http://www.tarduino.cc/2016/07/fixing-stuck-touchpad-on-toshiba.html
[6]: http://itexpertinbd.blogspot.tw/2014/11/email-id-sell-email-marketing-bd-id.html
[7]: http://enginemonitor.blogspot.tw/2016/09/becoming-embedded-programmer.html
[8]: http://www.myexperimentswithjava.com/2016/06/magic-with-computers.html
[9]: http://decimater.blogspot.tw/2016/09/driver-memory-leak.html
[10]: http://www.bluetrainsoftware.com/2014/11/polymer-without-dependency-injection-is.html